### DISCUSSION TITLE
使用find_package无法找到本地opencv库

### DISCUSSION THREAD

**Discussion by liaozixin** [xmake-io/xmake#2450] [Q&A]
使用cmake时可以成功通过find_package找到opencv库，但在xmake中会返回nil
`xmake l find_package cmake::opencv `返回`nil`

**Comment by waruqi** [comment]
你调下这里 https://github.com/xmake-io/xmake/blob/master/xmake/modules/package/manager/cmake/find_package.lua

**Reply by liaozixin** [reply]
谢谢您，已解决
